### DISCUSSION TITLE
info configurazione caddy

### DISCUSSION THREAD

**Discussion by Vincenzopica** [indomus/forum#3055] [Networking]
![Screenshot_20240428_135009_Home Assistant](https://github.com/indomus/forum/assets/101067938/ce30448f-11dd-4480-8187-e4987071c0bd)
Ciao a tutti, chiedo aiuto, sto cercando di configurare caddy come da guida, purtoppo non riesco a capire dove sbaglio, allego log di caddy e la risposta del broswer.
Grazie
![Screenshot_20240428_113731_Chrome](https://github.com/indomus/forum/assets/101067938/1309def8-abbd-44aa-b4af-7cb75bdf09a6)

**Comment by Scialla** [comment]
Devi postare anche il file di configurazione di caddy (dove presumo tu abbia sbagliato a configurare la TLS)

**Reply by Vincenzopica** [reply]
Grazie per la risposta.
Questo è nel file Caddy:
![Screenshot_20240428_152918_Home Assistant](https://github.com/indomus/forum/assets/101067938/e70d9793-e02e-4da5-8689-08bd7cbb83b5)



(https_header) {
  header {
    Strict-Transport-Security "max-age=31536000; includeSubdomains"
    X-XSS-Protection "1; mode=block"
    X-Content-Type-Options "nosniff"
    X-Frame-Options "SAMEORIGIN"
    Referrer-Policy "same-origin"
  }
}
https://miohome.duckdns.org {
  respond "Saluti dal Crew inDomus"
  file_server
}
https://miohome.duckdns.org:9000 {
  import https_header
  reverse_proxy http://192.168.1.15:8123
}

Poi nella configurazione:

![Screenshot_20240428_152719_Home Assistant](https://github.com/indomus/forum/assets/101067938/70e7c3d5-d44c-4733-94f6-df12e2126c7c)

**Reply by Scialla** [reply]
sul modem hai fatto il reindirizzamento della porta 9000 sull'ip del tuo HA?

Ah vedo che usi HAOS, lì non so aiutarti perché non l'ho mai usato

Leggi bene sulla guida quello che riguarda il port forwarding
https://indomus.it/guide/installare-e-configurare-caddy-su-home-assistant-os-hassos-hassio/

**Reply by Vincenzopica** [reply]
Queste sono le porte che ho aperto:

![Screenshot_20240428_153926_Chrome](https://github.com/indomus/forum/assets/101067938/bf81db16-ffb7-4a3c-8c5e-481332905d52)
![Screenshot_20240428_154052_Chrome](https://github.com/indomus/forum/assets/101067938/71292f85-27f8-4d19-89a9-1aac843c9210)

**Reply by Scialla** [reply]
sarebbe il caso imparassi ad usare il markdown postando le configurazioni e non le foto (incomplete)
Hai aperto le porte sull'IP dove gira HA? Hai fatto la prova senza il redirect per vedere se si connette? se non si connette hai letto il riquadro di cui ti dicevo prima? Devi capire tu se il tuo operatore e il tuo modem ti permettono di aprire la 443 e girarla sull'IP della macchina dove gira HA, porta necessaria per la TLS, oppure se sia il caso di utilizzare altro per i certificati

**Reply by Vincenzopica** [reply]
Si cercerò di capire cos'è il markdown, 
Si l'indirizzo ip è quello corretto, l'operatore è vodafone il modem/router è un fritz box 7590ax

**Reply by Vincenzopica** [reply]
Cosa intendi per la prova senza reindirizzamento?

**Reply by Scialla** [reply]
Che prima di inserire in config il reindirizzamento al tuo ip locale con il tuo FQDN avresti dovuto provare a far generare i certificati e doveva restituirti "Saluti dal crew inDomus"
https://indomus.it/guide/installare-e-configurare-caddy-su-home-assistant-os-hassos-hassio/#:~:text=Nei%20log%20dovremmo%20trovare%20qualcosa%20come%3A

**Reply by Vincenzopica** [reply]
Si quella parte funzionava,
![Screenshot_20240428_160452_Chrome](https://github.com/indomus/forum/assets/101067938/efc8f576-e623-4084-a9ef-4d43e4488b9a)

Anche adesso mi restituisce la stessa cosa se non metto la porta 8123

**Reply by Scialla** [reply]
8123? Ma non hai reindirizzato la 9000 sulla 8123? devi usare la 9000...
Devi avere fatto qualche pastrocchio sul modem, comunque ho provato e entro...
https://tuofqdn:8123/

Edit: stai modificando? perché ora da errore anche a me..

**Reply by Vincenzopica** [reply]
Ora sembra funzionare, mi erano sfuggite 2 righe in configuration.yaml, l'unica cosa ora, nella stessa rete non lo raggiungo più con l'indirizzo ip

**Reply by Scialla** [reply]
allora hai fatto altri casini... :D
http://192.168.1.15:8123 deve andare
cmq io cancellerei il topic perché ora tutti conoscono il tuo FQDN

Vabbé che con l'aggiornamento di maggio dovrebbero aggiungere il controllo sui device autorizzati..

**Reply by Vincenzopica** [reply]
Intanto, grazie continuo a lavorarci su.
Quindi cancello?

**Reply by Scialla** [reply]
se te lo fa ancora fare... togli i riferimenti al tuo FQDN

**Reply by Vincenzopica** [reply]
Ok l'ho modificato, ora non mi rimane che capire dove ho sbagliato per l'indirizzo interno

**Reply by Vincenzopica** [reply]
Ora da ip lo vedo così...

![Screenshot_20240428_180049_Chrome](https://github.com/indomus/forum/assets/101067938/19b9caa1-ab58-4f93-a230-17cebb2c04af)

**Reply by Scialla** [reply]
stai facendo casini... mi sa che non stai più usando caddy e hai abilitato let's encrypt sul configuration.yaml... ma sai solo tu cosa stai facendo.... ti suggerirei di ricominciare da capo seguendo pedessiquamente la guida

**Reply by Vincenzopica** [reply]
Si ho attivato quello, niente mi sa che non sono in grado... lascio via😟

**Reply by snakuzzo** [reply]
forse è meglio...visto che stai andando a caso
